interactive/interactive.go

```diff
@@ -35,6 +40,17 @@ func OpenInBrowser(url string) error {
 	return err
 }
 
+func WSL2() (bool, error) {
+	if runtime.GOOS != "linux" {
+		return false, nil
+	}
+	out, err := exec.Command("cat", "/proc/version").CombinedOutput()
+	if err != nil {
+		return false, errors.Wrapf(err, "detecting WSL2: %s", string(out))
+	}
+	return bytes.Contains(out, []byte("WSL2")), nil
```

Comment:
Do we need to check if it contains `WSL2` or `Microsoft`? Referriing to similar issue https://github.com/microsoft/WSL/issues/4071

Reply:
I'm just imagining someone having to come back to this code in 5 years like 'wth is a WSL?'

Reply:
Documented what is WSL at https://github.com/efficientgo/e2e/pull/57/commits/ad6f833585897a872b4ff5f2572335d5814d2ffe